beta/examples/tensorflow/object_detection/main.py

```diff
@@ -164,6 +171,36 @@ def train_step(dataset_inputs):
     return train_step
 
 
+def train_epoch(train_step, compression_ctrl, epoch, initial_epoch, steps_per_epoch, optimizer, checkpoint_manager,
+                train_dist_dataset, train_summary_writer, initial_step, print_freq, timer):
+    compression_ctrl.scheduler.epoch_step(epoch)
```

Comment:
probable we should mention that this function should contain NNCF stuff, like compression_scheduler.step()